### PR TITLE
02week pig latin

### PR DESCRIPTION
**Pull request by chrisdesilva**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] All tests passed: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript: 30 points

**Checkpoint 2**
- [ ] The application works as it should: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript and the DOM API: 30 points

**Checkpoint 3**
- [ ] Use of React: 25 points
- [ ] Accesses an API: 25 points
- [ ] Proper use of documentation (commenting on code): 25 points
- [ ] The application functions as it should: 25 points

### CODE REVIEW

File: 02week/pigLatin.js [hunk 1]
```diff
@@ -6,18 +6,59 @@ const rl = readline.createInterface({
...
 });
+//Create a variable to hold the array that converts all letters to lowercase and scrubs whitespace. Create a variable with an empty string that will be filled by for loop. Create a variable that lists all vowels.
 
+// (if) Write a function that runs if the word starts with a vowel, loop through array starting at index 0, then attach 'yay' to end. Create a variable to add letters to empty string variable.
+
+// (else if) Write a function that runs if the word starts with two consonants, loop through array starting at index 2, attach index 0 and 1, then 'ay' at end
+
+// (else if) Write a function that accepts a string. Convert string into array(split). Create a loop that runs through array starting at index 1 and ending at the end of the string. Attach index 0 to array (push). Attach 'ay' to array (push).
+
+// (else) run error message if input is not a string
```
Comment (stephsingler) [inline 02week/pigLatin.js]: great comments!

File: 02week/pigLatin.js [hunk 2]
```diff
@@ -6,18 +6,59 @@ const rl = readline.createInterface({
...
+  const strArray = word.toLowerCase().replace(/\s/g, '').split("");
+  // const vowel = ["a", "e", "i", "o", "u"];
+  const firstChar = strArray[0];
+  const secondChar = strArray[1];
+  let formattedString = "";
 
-  // Your code here
+  const startsWithVowel = () => {
+    for (var i = 0; i < strArray.length; i++)
+      formattedString += strArray[i];
```
Comment (stephsingler) [inline 02week/pigLatin.js]: need {} after your if statement.  Whatever you want to happen should go between them.

File: 02week/pigLatin.js [hunk 3]
```diff
@@ -6,18 +6,59 @@ const rl = readline.createInterface({
...
+  if (firstChar == 'a' || firstChar == 'e' || firstChar == 'i' || firstChar == 'o' || firstChar == 'u') {
+    return startsWithVowel()
+  } else if ((firstChar != 'a' || firstChar != 'e' || firstChar != 'i' || firstChar != 'o' || firstChar != 'u') && (secondChar == 'a' || secondChar == 'e' || secondChar == 'i' || secondChar == 'o' || secondChar == 'u')) {
+    return startsWithConsonant()
+  } else if ((firstChar != 'a' || firstChar != 'e' || firstChar != 'i' || firstChar != 'o' || firstChar != 'u') && (secondChar != 'a' || secondChar != 'e' || secondChar != 'i' || secondChar != 'o' || secondChar != 'u')) {
+    return startsWithDoubleConsonant()
+  } else {
+    return "error"
+  }
+
```
Comment (stephsingler) [inline 02week/pigLatin.js]: This entire if statement is not sitting inside a function. for this to work it needs to be inside something or you will get an error like "unknown: 'return' outside of function"

**Comment by chrisdesilva** [comment]
I made some changes, added, committed, and pushed, but now my workbook in Atom is back to where the code began with all of my new code erased. Not sure what happened.

**Comment by stephsingler** [comment]
Make sure you’re on the right branch!

**Pull request merged**